### PR TITLE
🐛 Fix Crash with useContentChecksums

### PR DESCRIPTION
**Pull request by mlch911**
### Description
Fix Crash with useContentChecksums.

### References
May crash when `cache` with useContentChecksums if there's any symlink file.
Shouldn't include these symlink when calculate checksums.

### Checklist
- [x] I've added at least one test that validates that my change is working, if appropriate
- [x] I've followed the code style of the rest of the project
- [x] I've read the [Contribution Guidelines](CONTRIBUTING.md)
- [x] I've updated the documentation if necessary

❤️ Thanks for contributing to the 🏈 Rugby!

### CODE REVIEW

**Comment by mlch911** [comment]
Fix #208

**Comment by swiftyfinch** [comment]
Thank you for your request!

File: Sources/Rugby/Commands/Cache/Other/Checksums/CombinedChecksum.swift [hunk 1]
```diff
@@ -102,7 +102,11 @@ extension LocalPod: CombinedChecksum {
         // 12% SHA1 checksum
         // 5% Other
         var checksums: [String] = []
-        for case let url as URL in enumerator where !url.hasDirectoryPath {
+        for case let url as URL in enumerator
+            where   !url.hasDirectoryPath &&
+                    !url.resolvingSymlinksInPath().hasDirectoryPath &&
+                    FileManager.default.isReadableFile(atPath: url.path)
+        {
```
Comment (swiftyfinch) [inline Sources/Rugby/Commands/Cache/Other/Checksums/CombinedChecksum.swift:109]: I think we need to resolve symlinks and after that to check `hasDirectoryPath` and `isReadableFile(atPath:)`. And it's a huge where block, let's put it as `guard` after `{` in for-loop.

```swift
for case let url as URL in enumerator {
    let resolvedSymlinksUrl = url.resolvingSymlinksInPath()
    guard FileManager.default.isReadableFile(atPath: resolvedSymlinksUrl.path),
          !resolvedSymlinksUrl.hasDirectoryPath else { continue }
    ...
}
```

Comment (mlch911) [reply Sources/Rugby/Commands/Cache/Other/Checksums/CombinedChecksum.swift:109]: Yes, this is better.

**Comment by swiftyfinch** [comment]
@mlch911 Please, fix `lint` warning.
As I see, there is an redundant space before `{`.

**Comment by mlch911** [comment]
Sorry, my fault.

**Pull request merged**